packages/cli/src/commands/generate.ts

```diff
@@ -7,14 +7,17 @@ import {
   FormGenerator,
   ModelGenerator,
   QueryGenerator,
+  singleCamel,
+  singlePascal,
+  pluralCamel,
+  pluralPascal,
```

Comment:
Food for thought for later.  Maybe it would be nice to create a new package for some of these util based files that can then be shared across other packages.  Right now we're exporting from the generator, but there's no reason these should be tied to that project.  🤷🏻

Reply:
The goal is to reduce the number of packages with a better build setup: https://github.com/blitz-js/blitz/issues/639